README.md

```diff
@@ -23,33 +23,75 @@ where:
 
 ## Configuration file
 
-The configuration file supports the following options:
+The configuration file supports the following top-level options:
 
- * `email = "<address>"` optionally specifies an email address to which any
-   errors running per-repo programs will be sent (warning: full stderr/stdout
-   will be sent, so consider carefully whether these have sensitive information
-   or not).
  * `maxjobs = <int>` is an (optional) non-zero integer specifying the maximum
    number of jobs to run in parallel. Defaults to the number of CPUs in the
    machine.
  * `port = <int>` is a mandatory port number to listen on (e.g. 4567).
+ * `github { ... }` specifies GitHub specific options.
```

Comment:
This is to allow for other (non-github) webhook senders to be integrated into snare?

Reply:
Correct.